apps/meeting/src/providers/AppStateProvider.tsx

```diff
@@ -53,11 +57,15 @@ export function AppStateProvider({ children }: Props) {
     }
   };
 
+  const toggleWebAudio = (): void  => {
+    setIsWebAudioEnabled(!isWebAudioEnabled);
```

Comment:
It is a good practice to use [functional updates](https://reactjs.org/docs/hooks-reference.html#functional-updates) to make sure to use correct "current" value when the next state is dependent on the previous state: 
```
setIsWebAudioEnabled(current => !current);
```